bigbluebutton/bigbluebutton
# fix(whiteboard): fit to width and zoom misc fixes

### What does this PR do?
Several fixes related to zoom and fit to width mode

- Ensure presenter & viewer camera are the same when using Fit To Width
- Some zoom issues when resizing/reloading
- Fix Viewer still sable to zoom with mouse wheel after someone takes presenter
- Do not apply shortcuts when using "Edit Link" shape dialog is open
- Hide Tldraw native reset zoom button as it does not to what expected
- Debounce zoomSlide messages to server, they were being set too frequently


### Motivation
Fit to width mode most of times did not sync the camera correctly between presenter * viewer, so they viwed different portions of slide.
Link edit shape option was not working because the typing was interrupted by shortcut handling.


### How to test
Join with 2 users, as presenter enable Fit To Width in toolbar, see if other users see the same area of slide, also zoom and pan to test.
